src/collections/_documentation/workflow/integrations/azure-devops.md

```diff
@@ -2,47 +2,79 @@
 title: Azure DevOps
 sidebar_order: 1
 ---
-Sentry’s new Azure DevOps integration, formerly known as Visual Studio Team Services (VSTS), has the following features: commit tracking, issue management, issue sync, and sign in with Azure Devops. You can now use the data from your commits to Azure DevOps to help you find and fix bugs faster.
+You can now use the data from your commits to Azure DevOps, formerly known as Visual Studio Team Services (VSTS), to help you find and fix bugs faster.
 
 ## Configure Azure DevOps
 
+{% capture __alert_content -%}
+Sentry owner or manager permissions, and Azure project admin permissions are required to install this integration.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+%}
 
-1. In Sentry, navigate to Organization Settings > Integrations. *Note: only users with Owner and Manager permissions will have access to this page.*
-2. If you have the legacy VSTS integration installed, you’ll see a button next to Azure DevOps that says ‘Upgrade’. If you do not have the legacy VSTS integration installed, you’ll see a button that says ‘Install.’ Click this button.
-3. In the resulting modal, click ‘Add Installation’.
-4. An Azure DevOps install window should pop up. Select the Azure DevOps account you'd like to link with Sentry, and press 'Submit.'
+1. In Sentry, navigate to Organization Settings > **Integrations**.
+2. If you have the legacy VSTS integration installed, you’ll see a button next to Azure DevOps that says **Upgrade**. If you do not have the legacy VSTS integration installed, you’ll see a button that says **Install**.
 
+    > [{% asset azure-global-install.png %}]({% asset azure-global-install.png @path %})
+
+3. In the resulting modal, click **Add Installation**.
+
+    > [{% asset azure-global-add-installation.png %}]({% asset azure-global-add-installation.png @path %})
+    
+4. An Azure DevOps install window should pop up. Select the Azure DevOps account you'd like to link with Sentry, and press **Submit**.
 
 Azure DevOps should now be enabled for all projects under your Sentry organization, but you'll need to configure the features below.
 
+> [{% asset azure-global-installed.png %}]({% asset azure-global-installed.png @path %})  
 
 ## Commit Tracking
 
-Commit tracking allows you to hone in on problematic commits. Learn more about commit tracking [here]({%- link _documentation/workflow/releases.md -%}#link-repository).
+Commit tracking allows you to hone in on problematic commits. [Learn more]({%- link _documentation/workflow/releases.md -%}#link-repository)
```

Comment:
Teeny nit - this looks a little odd/cut off  
https://cl.ly/b02fa353e30f
without the rest of the text